### ISSUE TITLE
Finetuing

### ISSUE THREAD

**Issue by xixinhy** [nadavbra/protein_bert#30]
Need your help, I want to know how to add a layer in the fine tuning?

**Comment by nadavbra** [comment]
You can see for example how we do that in [FinetuningModelGenerator](https://github.com/nadavbra/protein_bert/blob/master/proteinbert/model_generation.py).

**Comment by xixinhy** [comment]
hi，And if it’s feature extraction, what am I supposed to do?

**Comment by nadavbra** [comment]
I don't understand your question. Could you clarify?

**Comment by xixinhy** [comment]
> I don't understand your question. Could you clarify?
I want to put the fine-tuned get_model_with_hidden_layers_as_outputs output out separately, I don't know what to do

**Comment by xixinhy** [comment]
Thank you so much for helping me .
I want to do a sequence analysis.And I want to take the input vector for the Softmax layer ，which is the output of Local and Global respresentations.and use it as my feature extraction to feed it into my own framework.
